site/docs/hosting/vps.md

```diff
@@ -45,131 +120,486 @@ cd /etc/systemd/system
 2. Open your new service file with an editor:
 
 ```sh
-nano bot1.service
+nano <app-name>.service
 ```
 
+replacing `<app-name>` with any identifier.
+
+> `<app-name>.service` will be the name of your service.
+
 3. Add the following content:
 
 ```text
+[Unit]
+After=network.target
+
 [Service]
-ExecStart=<start_command>
+Environment=BOT_TOKEN=<token>
+Type=simple
+User=<username>
+ExecStart=<start-command>
 Restart=on-failure
 
 [Install]
 WantedBy=multi-user.target
 ```
```

Comment:
Should we add an explanation of this configuration?

Reply:
Hmmm yes, good idea